### PR TITLE
Enh/snake case

### LINKED ISSUE #27: Change camelCase variables to snake_case
### Summary
ICA-AROMA uses `camelCase` for variables, which is MATLAB convention and _not_ Python convention. We should switch to `snake_case`.

### Additional Detail
It _is_ Python convention to use UpperCamelCase for classes, but we don't define any in this package at the moment.

### Next Steps
1. Go through all files and replace camelCase variables with snake_case ones.

### PR DESCRIPTION
**Pull request by eurunuela**
Closes #27 .

Changes proposed in this pull request:

- Renames camelCase variables into snake_case format.

I hope I didn't miss any 🤞

### CODE REVIEW

**Comment by tsalo** [comment]
This looks great, thank you. The `dest` variables in the CLI will need to be updated as well though. Also, could you do the features file too?

**Comment by eurunuela** [comment]
Thought I added the features file to the commit. Sorry about that. Will commit now.

**Comment by eurunuela** [comment]
I think that should make it.

**Comment by tsalo** [comment]
The CLI target variables still need to be updated. Instead of the normal FSLDIR error, the integration test is failing on the following:

> TypeError: aroma_workflow() got an unexpected keyword argument 'denType'

**Comment by eurunuela** [comment]
That's weird. I did add the changes but for some reason, GitHub did not pick them up.

**Comment by eurunuela** [comment]
Okay, it should be fine now.

File: aroma/features.py [hunk 1]
```diff
@@ -57,7 +61,7 @@ def feature_time_series(melmix, mc):
         rp12[1:],
         np.zeros(2 * nparams)
     ))
-    rp_model = np.hstack((rp12, rp12_1fw, rp12_1bw))
+    rp_model = np.hstack((rp24, rp12_1fw, rp12_1bw))
```
Comment (tsalo) [inline aroma/features.py:64]: This reverts #32.

File: aroma/features.py [hunk 2]
```diff
@@ -48,6 +48,10 @@ def feature_time_series(melmix, mc):
     # Create an RP-model including the RPs and its derivatives
     rp12 = np.hstack((rp6, rp6_der))
 
+    # Add the squared RP-terms to the model
+    # NOTE: The above comment existed, but this step was **missing**!!
+    rp24 = np.hstack((rp12, rp12 ** 2))
+
```
Comment (tsalo) [inline aroma/features.py:54]: This reverts #32.

**Review by tsalo — CHANGES_REQUESTED** [review]
The core changes look good to me, but it looks like you're reintroducing some lines I changed in #32. I've flagged those lines with review comments.

**Comment by eurunuela** [comment]
> The core changes look good to me, but it looks like you're reintroducing some lines I changed in #32. I've flagged those lines with review comments.

Good catch, thank you!

**Review by tsalo — APPROVED** [review]
I think it'll be good to merge once you fix the merge conflict, so I'm approving.

**Pull request merged**